### ISSUE TITLE
How to pass sample_weigths to validation data?

### ISSUE THREAD

**Issue by rladeira** [fchollet/keras#3013]
In the method `fit` of a keras model, there is a parameter (`sample_weights`) which passes sample weights for training data. I think that is reasonable to also pass sample_weights for computing validation loss, so that overfitting can be tracked in these settings. Is this possible?

**Comment by fchollet** [comment]
Just pass as `validation_data` a triplet `(samples, labels, sample_weights)` instead of just `(samples, labels)`.
